src/e2e/pages/authPage.ts

```diff
@@ -39,16 +41,16 @@ export class AuthPage {
     await this.continue({ waitForURL: "**/oauth/**" });
   }
 
-  async enterPassword() {
-    await this.passwordInputField.fill(
-      process.env.E2E_TEST_ACCOUNT_PASSWORD as string,
-    );
+  async enterPassword(optionalPassword?: string) {
+    const password =
+      optionalPassword || (process.env.E2E_TEST_ACCOUNT_PASSWORD as string);
+    await this.passwordInputField.fill(password);
     await this.continue({ waitForURL: "**/user/**" });
   }
 
-  async signIn(email: string) {
+  async signIn(email: string, optionalPassword?: string) {
```

Comment:
where are we passing in this optional password?

Reply:
I use that in my forceLogin helper method